vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -154,33 +148,43 @@ ent prj install --conflict-strategy=OVERRIDE
 ```
 
 ### Run a Project Locally
-1. Initialize Keycloak, which leverages docker-compose
+1. Initialize Keycloak, which leverages Docker Compose
 ``` sh
 ent prj ext-keycloak start
 ```
 2. Initialize backend microservices
 ``` sh
 ent prj be-test-run
 ```
+To fetch logs from bundle plugins, use the command
+```
+ent prj be-log
+```
+
 3. Initialize one or more frontend widgets, each from its own shell
 ``` sh
 ent prj fe-test-run
 ```
-See [this tutorial](../../tutorials/create/ms/run-local.md) for more details.
+Check out [Run Blueprint-generated Microservices and Micro Frontends in Dev Mode](../../tutorials/create/ms/run-local.md) for more details.
+
+### Custom Resources
+Entando supports a series of commands to manage custom resources. To generate a custom resource and deploy it to the current profile, use `ent ecr deploy`. The subcommand `ent ecr cr` returns the usage status of a bundle. 
```

Comment:
The help text was misleading here. 
- The ecr commands manage bundles rather than simply custom resources
- The subcommand ent ecr cr actually generates the custom resource YAML for a bundle

I think we should add a new section for the ECR command, somehow I thought it already existed. Maybe it's enough to include these commands:  ent ecr list, ent ecr deploy, ent ecr get-plugin-id, and ent ecr get-bundle-id. I probably wouldn't link to the CR page from here.